tests/rkt_caps_test.go

```diff
@@ -90,14 +90,27 @@ func TestCaps(t *testing.T) {
 		defer os.Remove(stage2FileName)
 		stageFileNames := []string{stage1FileName, stage2FileName}
 
-		for _, stage := range []int{1, 2} {
+		var stages []int
+		if isKVM() {
```

Comment:
I am somehow becoming allergic towards those `if kvm` checks throughout the code. I am swimming in `stage1/init` now and it's cluttered there already, now creeping into tests.

What speaks against dedicated tests for kvm?